noaa/gefs/forecast/template.py

```diff
@@ -39,10 +47,24 @@ def get_template(init_time_end: DatetimeLike) -> xr.Dataset:
     ds["valid_time"].encoding = template_valid_time.encoding
     ds["valid_time"].attrs = template_valid_time.attrs
 
+    # Fill in expected_forecast_lengths based on init_time
+    template_expected_forecast_length = ds["expected_forecast_length"]
+    ds.coords["expected_forecast_length"] = (
+        "init_time",
+        [
+            INIT_TIME_HOURS_TO_FORECAST_LENGTHS[pd.Timestamp(init_time).hour]
+            for init_time in ds.init_time.values
+        ],
+    )
+    ds["expected_forecast_length"].encoding = template_expected_forecast_length.encoding
```

Comment:
We don't have any attrs yet, but lets copy them over too from the template in case we add them later